### PR TITLE
52 style moviemodal

### PR DESCRIPTION
**Pull request by marcusmoller97**
* Added style to movie modal to look like in the card.
* Remove exitButton div from modal.
* Made a git pull for  extra functionality from issue #61

### CODE REVIEW

**Review by NordicNomadicLife — APPROVED** [review]
looks good

File: __tests__/moviesJestTest.spec.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
+import '@testing-library/jest-dom';
+import jest from 'jest-mock';
+import { movieCard } from "../movies.js";
+
```
Comment (itsJoel59) [inline __tests__/moviesJestTest.spec.js:4]: Nice tests!

**Review by itsJoel59 — APPROVED** [review]
Nice work, looking forward to seeing the result!

**Review by kai-ericson — CHANGES_REQUESTED** [review]
Nice work! And good job with the tests. There is a merge conflict that needs to be fixed and then it's ready to merge.

**Review by kai-ericson — APPROVED** [review]

**Pull request merged**